tests/60app-services/06publicroomlist.pl

```diff
@@ -0,0 +1,238 @@
+use Future::Utils qw( try_repeat_until_success );
+
+
+use constant AS_PREFIX => "/_matrix/app/unstable";
+
+
+sub matrix_get_room_list_synced
```

Comment:
If you aren't going to export this, then maybe don't prefix it with ``matrix_``? I'm somewhat ambivalent on this though.